scripts/tests/test_zkai_minimal_transformer_block_benchmark_gate.py

```
@@ -0,0 +1,145 @@
+        self.assertEqual(rows["external_statement_receipt"]["object_class"], "external_snark_statement_receipt")
+        self.assertEqual(rows["native_full_block_proof_object"]["object_class"], "missing_native_proof_object")
+        self.assertIsNone(rows["native_full_block_proof_object"]["primary_value"])
+        self.assertEqual(rows["nanozk_context_row"]["comparability"], "CONTEXT_ONLY_NOT_LOCAL_REPRODUCTION_NOT_MATCHED_WORKLOAD")
+        self.assertEqual(rows["gkr_hyrax_sidecar_lane"]["local_status"], "FOLLOWUP_ISSUE_650_NOT_IMPLEMENTED")
+        self.assertEqual(rows["jolt_atlas_lookup_tensor_lane"]["local_status"], "FOLLOWUP_ISSUE_651_NOT_IMPLEMENTED")
+
+    def test_rejects_component_omission(self) -> None:
+        payload = gate.build_payload()
+        payload["component_rows"].pop(0)
+        payload["payload_commitment"] = gate.payload_commitment(payload)
+        with self.assertRaises(gate.MinimalBlockBenchmarkError):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)